### DISCUSSION TITLE
justify-content: left(right)

### DISCUSSION THREAD

**Discussion by pisyell** [csstools/postcss-plugins#897] [Q&A]
how to fix ?

**Comment by Antonio-Laguna** [comment]
@pisyell could you provide more information?

**Reply by pisyell** [reply]
Will justify-content:right  be converted to justify-content:flex-end?

**Reply by Antonio-Laguna** [reply]
No, it would not, there's no plugin for that yet.

**Reply by romainmenke** [reply]
There was a request for this a while back and I checked this with the CSSWG (specification authors) but there isn't any transform that would be correct : https://github.com/csstools/postcss-plugins/issues/108#issuecomment-1398984469

All keywords for `justify-content` mean different things.
`left` and `right` are not modern replacements for `flex-start` and `flex-end`.

**Reply by pisyell** [reply]
ok thanks
